bin/burden

```diff
@@ -2507,7 +2566,7 @@ run_scenario()
 				pindex=0
 				pid=""
 			else
-				tmpfile=$(mktemp /tmp/zath_temp.XXXXXX)
+				tmpfile=$(mktemp /tmp/zath_temp4.XXXXXX)
```

Comment:
What's the purpose of `zath_temp[0-9].XXXXX`?

Name does not convey the intent of the file